<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QWidget" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="LogInPushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="IdentityComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>普通用户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理员</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;ID/CNO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserNameContent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordContent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
